lib/themes/dosomething/paraneue_dosomething/js/Analytics.js

```diff
@@ -29,6 +29,17 @@ define(["validation", "modal"], function(Validation, Modal) {
       ga("send", "event", "Form", "Validation Error on submit", args);
     });
 
+    $(".js-analytics-fb-share").on("click", function() {
```

Comment:
Can you add an empty newline between each block, and a little `// Confirmation Page Shares` header above this section? :neckbeard: